cernopendata_client/printer.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# cernopendata-client is free software; you can redistribute it and/or modify
+# it under the terms of the GPLv3 license; see LICENSE file for more details.
+
+"""cernopendata-client output print configuration."""
+
+import click
+
+
+def display_message(prefix=None, msg_type=None, msg=None):
+    """Display message in a similar style as run_command().
+
+    :param prefix: prefix to be added in output (==> / ->)
+    :param msg_type: Type of message - Error/Progress/Success
+    :param msg: message to display
+    :type prefix: str
+    :type msg_type: str
+    :type msg: str
+    """
+    msg_color_map = {
```

Comment:
This could be moved to the global `config.py` now that we have it, e.g.

```python
PRINTER_COLOUR_INFO='green'
PRINTER_COLOUR_ERROR='red'
```

Reply:
Done 👍

Reply:
Done 👍